pocs/utils/theskyx.py

```diff
@@ -56,6 +56,7 @@ def read(self, timeout=5):
 
             try:
                 response = self.socket.recv(4096).decode()
+                self.logger.debug('Response from reading TSX socket: {}'.format(response))
```

Comment:
This should be removed. I was using it for debug and it generated way too much output.

Reply:
Done.